chaining/challenges-10.test.js

```diff
@@ -0,0 +1,190 @@
+
+'use strict';
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named count that, given an integer and an array of arrays, uses either
+// filter, map, or reduce to count the amount of times the integer is present in the array of arrays.
+//
+// Note: You might need to use the same method more than once.
+//
+// For example, count(5, [[1, 3, 5, 7, 9], [5, 5, 5], [1, 2, 3]]) returns 4.
+// ------------------------------------------------------------------------------------------------
+
+const count = (target, input) => {
+  // Solution code here...
+  let amountOfTimesIntegerPresent = [];
+  input.map(num => {
+    num.forEach(n => {
+      if (n === target) {
+        amountOfTimesIntegerPresent.push(n);
```

Comment:
It might make more sense to count this using a number, rather than pushing things into an array! You're filling in that array, which isn't useful to you. So something like `amountOfTimes++` here, instead of pushing into an array.